src/views/MissionPlanningView.vue

```diff
@@ -1259,21 +1377,21 @@ const createSurveyVertexMarker = (
   return L.marker(latlng, {
     icon: L.divIcon({
       html: `
-        <div class="survey-vertex-icon">
-          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <circle cx="12" cy="12" r="5" fill="#3B82F6" stroke="white" stroke-width="2"/>
-          </svg>
-          <div class="delete-popup" style="display: none;">
-            <button class="delete-button">
-              <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path d="M2 4h12M4 4v10a2 2 0 002 2h4a2 2 0 002-2V4M6 4V2h4v2" 
-                      stroke="white" stroke-width="1.5" 
-                      stroke-linecap="round" stroke-linejoin="round"/>
-                </svg>
-              </button>
+          <div class="survey-vertex-icon">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <circle cx="12" cy="12" r="5" fill="#3B82F6" stroke="white" stroke-width="2"/>
+            </svg>
+            <div class="delete-popup" style="display: none;">
+              <button class="delete-button">
+                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M2 4h12M4 4v10a2 2 0 002 2h4a2 2 0 002-2V4M6 4V2h4v2" 
+                        stroke="white" stroke-width="1.5" 
+                        stroke-linecap="round" stroke-linejoin="round"/>
+                  </svg>
+                </button>
+            </div>
           </div>
-        </div>
-      `,
+        `,
```

Comment:
No need to reindent.

It looks like a lot of leftovers from the mission planning PR were readded here.